### ISSUE TITLE
Request: Compound identification database import/export in mzTab

### ISSUE THREAD

**Issue by photocyte** [mzmine/mzmine3#84]
Right now mzMine2 can import custom metabolite ID databases in CSV format.  This is flexible, but why not use mzTab format explicitly (which I believe has a field for identifications) as the format for importing/exporting compound identifications?  Ideally the mzTab / compound identification file could also have parseable InChI for the feature in question, which would allow computation of parameters.

**Comment by dyrlund** [comment]
InChI and InChIKeys is something that we have also discussed to support for CSV files: #52.

**Comment by photocyte** [comment]
Thinking about this more, I agree that mzTab is not the best choice for the main format for the identification database, as it makes the the difference between peaklists (quantification) and identification, less clear.  That being said, it would be a useful feature to modify the CSV import such that it could handle the mzTab files without modification (ignore multiple header lines, ignore COM lines etc.).  Will attempt implementing in MZmine2

**Comment by dyrlund** [comment]
I agree. Please also look at how we have made the implementation for the local database search in MZmine 3. You could update the parameter dialog so that it can get the annotations from the mzTab files and then write them to the text field.
